reana-ui/src/components/WorkflowDeleteModal.js

```diff
@@ -38,9 +38,9 @@ export default function WorkflowDeleteModal() {
           <Message.Content>
             <Message.Header>Workspace deletion!</Message.Header>
```

Comment:
What do you think about changing also the header to something like "Deletion of workspace and interactive sessions!"?